src/core/jsonpointer/include/sourcemeta/core/jsonpointer_pointer.h

```diff
@@ -1,14 +1,14 @@
 #ifndef SOURCEMETA_CORE_JSONPOINTER_POINTER_H_
 #define SOURCEMETA_CORE_JSONPOINTER_POINTER_H_
 
+#include <cstddef>
```

Comment:
Most headers follow a specific header ordering: the Sourcemeta headers followed by the standard include headers. Can you respect that ordering? You can put `cstddef` below, and also as a comment put why you are including it.

Reply:
Is there a reason for this order?

Reply:
Just convention. Otherwise the headers are all over the place